src/openlcb/ServoConsumer.hxx

```diff
@@ -0,0 +1,110 @@
+#ifndef _OPENLCB_SERVOCONSUMER_HXX_
+#define _OPENLCB_SERVOCONSUMER_HXX_
+
+#include "freertos_drivers/common/DummyGPIO.hxx"
+#include "freertos_drivers/common/PWM.hxx"
+#include "openlcb/ServoConsumerConfig.hxx"
+#include "os/MmapGpio.hxx"
+#include <memory>
+
+namespace openlcb
+{
+
+// Basically a specialized ConfiguredConsumer.
```

Comment:
please use doxygen style comments, i.e. `///` for field and structure descriptions. Make sure the comment would read meaningful in the documentation as well.